kotlinx-coroutines-core/jvm/test/MemoryFootprintTest.kt

```diff
@@ -5,14 +5,32 @@
 package kotlinx.coroutines
 
 import org.junit.Test
-import org.openjdk.jol.info.ClassLayout
+import org.openjdk.jol.info.*
 import kotlin.test.*
 
 
 class MemoryFootprintTest : TestBase(true) {
```

Comment:
My intention was not "test the property of the code here".

`Job` is, more or less, quite footprint-sensitive (e.g. is a significant contributor to the coroutines memory footprint).
I thought it would be beneficial to keep track of the footprint in a manner similar to our `testJsMiscompilationIsStillHappening` -- so we would know if anything changed and can be sure it is a deliberate decision.

I also believe it's going to be a useful quality assurance when migrating to new CLL, and putting it into integration tests is going to make use of it much harder

Reply:
Got it, thanks!